src/commands/bans/banned.ts

```diff
@@ -0,0 +1,23 @@
+import {Command} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+import {getBannedWord} from "../../controllers/BanController";
+
+const banned: Command = {
+    name: "banned",
+    description: "Lists the banned word (if any)",
+    usage: "banned",
+    procedure: async (client: Client, message: Message) => {
+        const banned_word = await getBannedWord();
+        let reply;
+        if (banned_word) {
+            reply = "The current banned word is: ||";
+            reply += banned_word;
+            reply += "||";
```

Comment:
Maybe use string interpolation?

```ts
reply = `The current banned word is: || ${bannedWord} ||`
```

Reply:
Might be a stylistic thing (I know @braxtonhall) doesn't like ternary operators, but the conditional could be simplified to
```ts
const reply = bannedWord ? `The current banned word is: || ${bannedWord} ||` : "No word is banned at the moment";
```
removing the need for mutation.

Reply:
Cool! That is a lot nicer to use.

Reply:
I'll put the ternary on a separate commit so it can be rolled back if it's unwanted. I generally avoid ternaries but this case is simple enough (and reducing mutability is often good).